examples/plot_postgres_create.py

```diff
@@ -228,8 +231,8 @@
 CREATE TABLE "sensor" (
     "sensor_id" VARCHAR(255) NOT NULL,
     "temporal_res" FLOAT NOT NULL,
-    "spatial_res_x" FLOAT NOT NULL,
-    "spatial_res_y" FLOAT NOT NULL,
+    "spatial_res" FLOAT NOT NULL,
```

Comment:
this should be fixed in dbdesigner. It's `spatial_res_x` and `spatial_res_y` now